test/library/standard/Endian/Endian.chpl

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
This file needs a corresponding `.notest` file with the same basename (aka `Endian.notest`).  Otherwise start_test tries to run it when running the directory it lives in

Reply:
Fixed it in 180ddb1